sources/assets/zsh/zshrc

```diff
@@ -84,3 +84,15 @@ else
   [ -d /opt/my-resources/setup/zsh ] || mkdir -p /opt/my-resources/setup/zsh
   cp /.exegol/skel/zsh/zshrc /opt/my-resources/setup/zsh/zshrc
 fi
+
+fzf-haiti () {
+    haiti -e "$1" | fzf --ansi | grep -E -o ' [0-9]*]' | cut -d ']' -f 1 | cut -d ' ' -f 2
```

Comment:
This should be set as an alias in `sources/assets/zsh/aliases.d/fzf` right after `fzf-wordlists`